registration/settings.py

```diff
@@ -54,7 +55,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': ['/home/saad/mysite/registration/it_fest/templates/it_fest',],
```

Comment:
Do you think its appropriate?
Can you explain why you have taken this approach?